tests/infrastructure/tls-configuration.go

```diff
@@ -49,38 +49,35 @@ import (
 
 var _ = DescribeInfra("tls configuration", func() {
 
-	var (
-		virtClient kubecli.KubevirtClient
-		cipher     *tls.CipherSuite
-	)
+	var virtClient kubecli.KubevirtClient
+
+	// FIPS-compliant so we can test on different platforms (otherwise won't revert properly)
+	cipher := &tls.CipherSuite{
+		ID:   tls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
+		Name: "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+	}
 
 	BeforeEach(func() {
 		virtClient = kubevirt.Client()
 
-		if !checks.HasFeature(virtconfig.VMExportGate) {
-			Skip(fmt.Sprintf("Cluster has the %s featuregate disabled, skipping  the tests", virtconfig.VMExportGate))
-		}
-
-		// FIPS-compliant so we can test on different platforms (otherwise won't revert properly)
-		cipher = &tls.CipherSuite{
-			ID:   tls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
-			Name: "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
-		}
 		kvConfig := util.GetCurrentKv(virtClient).Spec.Configuration.DeepCopy()
 		kvConfig.TLSConfiguration = &v1.TLSConfiguration{
 			MinTLSVersion: v1.VersionTLS12,
 			Ciphers:       []string{cipher.Name},
 		}
 		tests.UpdateKubeVirtConfigValueAndWait(*kvConfig)
-		newKv := util.GetCurrentKv(virtClient)
-		Expect(newKv.Spec.Configuration.TLSConfiguration.MinTLSVersion).To(BeEquivalentTo(v1.VersionTLS12))
-		Expect(newKv.Spec.Configuration.TLSConfiguration.Ciphers).To(BeEquivalentTo([]string{cipher.Name}))
-
 	})
 
 	It("[test_id:9306]should result only connections with the correct client-side tls configurations are accepted by the components", func() {
-		labelSelectorList := []string{"kubevirt.io=virt-api", "kubevirt.io=virt-handler", "kubevirt.io=virt-exportproxy"}
+		By("Verify Kubevirt has correct configuration")
+		newKv := util.GetCurrentKv(virtClient)
+		Expect(newKv.Spec.Configuration.TLSConfiguration.MinTLSVersion).To(BeEquivalentTo(v1.VersionTLS12))
+		Expect(newKv.Spec.Configuration.TLSConfiguration.Ciphers).To(BeEquivalentTo([]string{cipher.Name}))
 
+		labelSelectorList := []string{"kubevirt.io=virt-api", "kubevirt.io=virt-handler"}
+		if checks.HasFeature(virtconfig.VMExportGate) {
+			labelSelectorList = append(labelSelectorList, "kubevirt.io=virt-exportproxy")
+		}
```

Comment:
Why did you moved this to the test itself? All these steps seem to be a pre-requisite of this test, not part of the test itself. It is often useful to separate the two, knowing if it is a test setup issue or the functionality issue.

Reply:
I moved this so the test can run in any configuration, with or without the feature gate. I expect us to use labels for skipping not the skip we have right now.

Reply:
Well, I actually wanted to drop the KV check, seems redundant, wdyt?

Reply:
I agree, I will drop it for now.